lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -2079,8 +2069,7 @@ def set_payload(resource_type, record, entry, is_json)
       # 2. The only remaining key is 'message'
       if is_json
         json_payload = record
-      elsif [GKE_CONSTANTS[:resource_type],
-             DOCKER_CONSTANTS[:resource_type]].include?(resource_type) &&
+      elsif GKE_CONSTANTS[:resource_type] == resource_type &&
             record.key?('log')
```

Comment:
These should now fit on one line, right?

Reply:
Done.